modules/uhsg_obar/uhsg_obar.module

```diff
@@ -26,7 +26,7 @@ function uhsg_obar_library_info_build() {
  * Implements hook_preprocess_hook().
  */
 function uhsg_obar_preprocess_region(&$variables) {
-  if ($variables['region'] == 'header' && uhsg_obar_is_valid_obar_route_and_domain()) {
+  if (($variables['region'] == 'header' || $variables['region'] == 'footer') && uhsg_obar_is_valid_obar_route_and_domain()) {
```

Comment:
This:

```php
($variables['region'] == 'header' || $variables['region'] == 'footer')
```

Could have a bit less code duplication and also be a bit shorter like this:

```php
in_array($variables['region'], ['header', 'footer'])
```

Reply:
Roger that. Added those changes. Thanks :)